### PR TITLE
Adding dependency StorageClass

### PR DESCRIPTION
**Pull request by cccsss01**
Adding dependency storage class to production README.md I forgot this was required.

### CODE REVIEW

File: examples/production/README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Recommended production setup
 
+Dependencies
+```
+StorageClass
+```
```
Comment (marcofranssen) [inline examples/production/README.md:6]:
```suggestion

- StorageClass
```

**Comment by kfox1111** [comment]
I think this pr's good. But raises a further question. With the new docs in the chart at  https://artifacthub.io/packages/helm/spiffe/spire#production

and similar docs lined up for the spiffe website itself, should we change this file to just point at the other docs? Its kind of out of sync with those recommendations.

**Comment by cccsss01** [comment]
Sounds good to me.

**Comment by cccsss01** [comment]
On second thought I think i'd like this open, and likely add a disclaimer stating to check other pages.

**Comment by cccsss01** [comment]
How is artifacthub documentation version controlled?

**Comment by kfox1111** [comment]
The artifacthub page is just displaying the README.md out of the charts. So in some ways its more relevant then whats in the git repo, as it is the README.md associated with the particular version of the chart.

**Comment by kfox1111** [comment]
Filed https://github.com/spiffe/helm-charts-hardened/issues/217 for more discussion

**Comment by faisal-memon** [comment]
@cccsss01 thanks for taking the time for this. Our plan is to delete the `examples/` folder as its fallen out of date so we dont need to update this readme anymore.

**Comment by kfox1111** [comment]
examples/production specifically.

**Pull request closed without merging**